cpp/src/arrow/flight/sql/odbc/tests/dremio/set_up_dremio_instance.sh

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#!/bin/bash
+set -e
+
+HOST_URL="http://localhost:9047"
+NEW_USER_URL="$HOST_URL/apiv2/bootstrap/firstuser"
+LOGIN_URL="$HOST_URL/apiv2/login"
+SQL_URL="$HOST_URL/api/v3/sql"
+
+ADMIN_USER="admin"
+ADMIN_PASSWORD="admin2025"
+
+# Wait for Dremio to be available.
+until curl -s "$NEW_USER_URL"; do
+    echo 'Waiting for Dremio to start...'
+    sleep 5
+done
+
+echo ""
+echo 'Creating admin user...'
+
+# Create new admin account.
+curl -X PUT "$NEW_USER_URL" \
+     -H "Content-Type: application/json" \
+     -d "{ \"userName\": \"$ADMIN_USER\", \"password\": \"$ADMIN_PASSWORD\" }"
+
+echo ""
+echo "Created admin user."
+
+# Use admin account to login and acquire a token.
+TOKEN=$(curl -s -X POST "$LOGIN_URL" \
+     -H "Content-Type: application/json" \
+     -d "{ \"userName\": \"$ADMIN_USER\", \"password\": \"$ADMIN_PASSWORD\" }" \
+     | grep -oP '(?<="token":")[^"]+')
+
+echo "TOKEN=$TOKEN"
```

Comment:
For security purpose, should we remove this line?

Reply:
Done.